package.json

```diff
@@ -90,5 +96,8 @@
     "ts-jest": "27.0.3",
     "ts-node": "~9.1.1",
     "typescript": "~4.3.5"
+  },
+  "volta": {
+    "node": "14.17.5"
```

Comment:
Suggestion: Add a comment similar to line 5 since this version should also be in sync.

Reply:
Good point!